drivers/pwm/Kconfig.nrf5_sw

```diff
@@ -19,28 +19,9 @@ if PWM_NRF5_SW
 
 config PWM_NRF5_SW_0_DEV_NAME
 	string "Nordic Semiconductor nRF5x series S/W PWM Device Name"
-	default "PWM_0"
+	default "SW_PWM"
```

Comment:
Changed to avoid confusion between HW PWM driver and SW PWM driver. Is there a mechanisms to handle multiple drivers being labeled the same thing?